drivers/hmis/lib/tasks/one_time_migration_20240303.rake

```diff
@@ -74,8 +98,8 @@ class OneTimeMigration20230303
     value = value =~ /\A[0-9]+\z/ ? "q_#{value}" : value
     # leading number with a period seems to be HUD ref numbers like "2.02.6"
     value = value =~ /\A[^a-z]+\./i ? "hud_#{value}" : value
-    # camelize hyphenated strings or spaces
-    value = value.gsub(/(-| )+([a-z])/) { Regexp.last_match(2).upcase }
+    # snakecase hyphenated strings or spaces
+    value = value.gsub(/(-| )+/, '_')
     # replace dots and lingering hyphens with an underscore
     value = value.gsub(/[-.]/, '_')
```

Comment:
I felt like it was unnecessarily inconsistent to camelcase some and snake case others, so just made it snake case on both. That ok?